### ISSUE TITLE
permission denied for database "postgres"

### ISSUE THREAD

**Issue by grauschnabel** [adaptivdesign/odooku#7]
Hi there,

I updated odooku today to get the newest odoo from git.  but now i run into the trouble of an internal server error.  looking at the heroku logs I see
```
2017-01-21T19:54:16.014298+00:00 app[web.1]: OperationalError: FATAL:  permission denied for database "postgres"
2017-01-21T19:54:16.014299+00:00 app[web.1]: DETAIL:  User does not have CONNECT privilege.

```
So but why does odoo now want to connect to postgres, because you never have permissions if you use heroku-posgres.  they put many user databases in one amazon postgres server, so it should be enaugh to have one database, and it is correctly set as a DATABASE_URL (I didn't change it and i can connect it with heroku pg:psql

So how can I make odoo work again?

Thanks,
Martin

**Comment by adaptivdesign** [comment]
Hey,

I've just run an instance on Heroku through the deploy button, just to make sure everything works as expected. Right now it does. It runs in single db mode. I did just fix something that could have caused your problem.

I recently changed a lot, so your problem may be caused because of a corrupt build. To get this fixed:

- (I assume you are running the 10.0 branch). If not I highly recommend it, as I'm not going to be supporting the 9.0 branch.
- Recreate or clean your Heroku app. See: https://github.com/heroku/heroku-repo .. use the repo:purge_cache command.
- Make sure you have a requirements.txt file with `git+https://github.com/adaptivdesign/odooku-compat.git@10.0` in it. (This has changed a few days ago)
